modulefiles/gsi_gaea.intel.lua

```diff
@@ -1,42 +1,23 @@
 help([[
 ]])
 
-unload("intel")
-unload("cray-mpich")
-unload("cray-python")
-unload("darshan")
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
```

Comment:
is this really needed? (I think gsi-addon-dev provides everything)

Reply:
@jswhit I tested with just gsi-addon-dev load and don't see any issues. I committed an updated module file.